screens/FindGamePage.js

```diff
@@ -0,0 +1,251 @@
+import React from 'react';
+import { StyleSheet, Text, View, Image, FlatList, Button } from 'react-native';
+import { Font } from 'expo';
+import { StackNavigator } from 'react-navigation';
+
+export class FindScreen extends React.Component {
+  state = {
+    fontLoaded: false,
+  };
+  async componentDidMount() {
+    await Font.loadAsync({
+      'double-bubble-shadow': require('./../assets/fonts/Double_Bubble_shadow.otf'),
+      'source-sans-pro': require('./../assets/fonts/source-sans-pro.semibold.ttf'),
+      'source-sans-pro-bold': require('./../assets/fonts/SourceSansPro-Bold.ttf'),
+    });
+    this.setState({ fontLoaded: true });
+  }
+  render() {
+    return (
+      <View style = { styles.container }>
+        <View style = { styles.find }>
+          <Text style = { this.state.fontLoaded ? styles.findText : styles.findTextElse }>FIND</Text>
+          <Text style = { this.state.fontLoaded ? styles.findText : styles.findTextElse }>A GAME</Text>
+        </View>
+        <View style = {{ paddingTop: '7.5%' }}>
+          <View style = { styles.time }>
+            <View style = { styles.timeTop }>
+              <Image style = {{ height: 40, width: 40 }} source={require('./../assets/time.png')}/>
+              <Text style = { this.state.fontLoaded ? styles.headerText : styles.headerTextElse }>Time</Text>
+            </View>
+            <View style = {{ height: 3, width: '100%', backgroundColor: '#fff' }}>
+            </View>
+            <View style = { styles.timeBottom }>
+              <View style = { styles.optionButtons }>
+                <Text style = { this.state.fontLoaded ? styles.optionsTimeText : styles.optionsTimeTextElse }>Now</Text>
```

Comment:
`styles.optionsTimeTextElse` isn't defined. Either remove this still or define the style